test/e2e/e2e_test.go

```diff
@@ -109,6 +108,7 @@ func init() {
 
 func TestMain(m *testing.M) {
 	rand.Seed(time.Now().UnixNano())
+	flag.Parse()
```

Comment:
the comment on line 82 is wrong after this change.

Reply:
can we move the whole call to `handleFlags()` to `TestMain` instead?

Reply:
we need to move **everything** in `init` to here AFAICT

Reply:
init is doing more with the flags after `handleFlags`